third_party/ibis/ibis_teradata/client.py

```diff
@@ -66,7 +66,7 @@ class TeradataClient(SQLClient):
     table_class = TeradataTable
     dialect = compiler.TeradataDialect
 
-    def __init__(self, host, user_name, password, port=1025, use_no_lock_tables=False):
+    def __init__(self, host, user_name, password, port=1025, use_no_lock_tables=False, logmech):
```

Comment:
If you go with my suggestion to put the logmech after the port, this will need to be re-ordered like so:
`def __init__(self, host, user_name, password, port=1025, logmech="TD2", use_no_lock_tables=False):`

Reply:
Got it! updated in the new commit